test/controllers/quote_controller_test.exs

```diff
@@ -44,7 +44,52 @@ defmodule HonGraffitiPhoenix.QuoteControllerTest do
 
     assert json_response(conn, 422) ==
       render_changeset_json("error.json", changeset: invalid_quote_changeset)
-    refute Repo.get_by(Quote, raw: params_for(:quote).raw)
+    refute Repo.get_by(Quote, raw: params_for(:invalid_quote).raw)
+  end
+
+  test "#update renders the updated quote and persists a valid quote" do
+    conn = build_conn()
+    original_quote_attrs = %{raw: "^gOriginal quote"}
+    updated_quote_attrs = %{raw: "^yUpdated quote"}
+    original_quote = insert(:quote, original_quote_attrs)
+
+    conn = patch(
+      conn,
+      quote_path(conn, :update, original_quote),
+      quote: params_for(:quote, updated_quote_attrs)
+    )
+
+    updated_quote = Repo.get_by(Quote, raw: updated_quote_attrs.raw)
+    assert json_response(conn, 200) ==
+      render_json("show.json", quote: updated_quote)
+    refute Repo.get_by(Quote, original_quote_attrs)
```

Comment:
Fair question... I think my rationale was that I'm already asserting that the new value is in the database by asserting that the JSON response contains the updated quote info, so by refuting the existence of the old one in the database I'm sort of testing the counterpoint to that? Idk, I guess maybe it would be more direct to just assert the new quote's existence.